app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -739,4 +742,11 @@ object TelemetryWrapper {
     fun addSearchEngineLearnMoreEvent() {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.ADD_SEARCH_ENGINE_LEARN_MORE).queue()
     }
+
+    private fun isDistributedDevice(): Boolean {
```

Comment:
tiny nit: We may not want to disable this for all distribution, so I wonder if we could make the method name more explicit... something like isDeviceWithTelemetryDisabled()?

Reply:
Sounds good to me